client/apollo/css/src/Form/Select/SelectCommon.scss

```
@@ -0,0 +1,75 @@
+@use "../../common/reboot";
+@use "../../common/tokens";
+
+.af-form__select {
+  &-label {
+    display: inline-block;
+    margin-bottom: calc(8 / var(--font-size-base) * 1rem);
+    font-size: calc(18 / var(--font-size-base) * 1rem);
+    font-weight: 600;
+    color: var(--dropdown-placeholder-color-default);
+  }
+
+  &-input {
+    display: block;
+    width: var(--dropdown-width);
+    margin: calc(10 / var(--font-size-base) * 1rem) 0;
+    padding: calc(21 / var(--font-size-base) * 1rem);
+    border: 1px solid var(--dropdown-border-color-default);
+    border-radius: var(--radius-4);
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    color: var(--dropdown-placeholder-color-default);
+    background: var(--dropdown-background);
+    background-color: white;
+    background-position-x: calc(100% - 1rem);
+    appearance: none;
+
+    option {
+      color: var(--axa-gray-20);
+
+      &:disabled {
+        border: 1px solid var(--gray-500);
+      }
+    }
+
+    &:open {
+      background-image: var(--dropdown-background-open);
```

le composant s'appelle "select"
```suggestion
      background-image: var(--select-background-open);
```

---

J'ia utilisé dropdown suite a référnece dans les maquettes foundations: https://www.figma.com/design/AhUJ7ZDUwSEaxD5pKaN3hq/Fondations-B2C?node-id=9068-60287&t=sLZ3uHojXvPiVzoq-0 , Je peux utiliser le select mais je te laisse trancher pour ca

---

j'ai vu avec les UX, pour le moment, on reste sur le nom Select car c'est du natif
Si plus tard, on créé un composant plus élaboré, on l'appellera Dropdown